package.json

```diff
@@ -85,86 +80,90 @@
     "react-sortable-hoc": "^1.10.1",
     "reactcss": "^1.2.3",
     "robust-point-in-polygon": "^1.0.3",
-    "slugid": "^1.1.0",
+    "slugid": "^3.0.0",
+    "stream": "0.0.2",
     "threads": "^0.7.2",
-    "transform-loader": "^0.2.3",
-    "unminified-webpack-plugin": "^2.0.0",
+    "transform-loader": "^0.2.4",
+    "unminified-webpack-plugin": "^3.0.0",
     "url-join": "^1.1.0",
     "url-parse": "^1.4.3",
     "vkbeautify": "^0.99.3",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.6.2"
   },
   "devDependenciesComments": [
     "webpack-dev-server is pinned because later versions did not serve all static directories: https://github.com/higlass/higlass/issues/623"
   ],
+  "overrides": {
+    "cheerio": "1.0.0-rc.10"
+  },
   "devDependencies": {
-    "ajv-cli": "^3.0.0",
-    "autoprefixer": "^7.2.6",
-    "babel": "^6.23.0",
-    "babel-core": "^6.26.3",
+    "@babel/core": "^7.15.8",
+    "@babel/plugin-proposal-object-rest-spread": "^7.15.6",
+    "@babel/plugin-transform-runtime": "^7.15.8",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.15.8",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/runtime": "^7.15.4",
+    "ajv-cli": "^5.0.0",
+    "autoprefixer": "^10.4.2",
     "babel-eslint": "^10.0.1",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-react-css-modules": "^3.4.2",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-airbnb": "^2.6.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.11.1",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-react-css-modules": "^5.2.6",
+    "babel-preset-airbnb": "^5.0.0",
     "bootstrap": "^3.3.7",
-    "brfs": "1.4.3",
+    "brfs": "^2.0.2",
     "browser-sync": "^2.24.7",
-    "chai": "^3.5.0",
-    "css-loader": "^0.28.8",
-    "del": "~2.2.0",
+    "chai": "^4.3.4",
+    "css-loader": "^3.6.0",
+    "css-minimizer-webpack-plugin": "^3.4.1",
+    "del": "^6.0.0",
     "enzyme": "^3.4.1",
-    "es6-promise": "~3.1.2",
-    "eslint": "^5.8.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-prettier": "^6.9.0",
+    "es6-promise": "^4.2.8",
+    "eslint": "^8.7.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "7.11.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.14.2",
     "fast-css-loader": "^1.0.2",
```

Comment:
It looks like `fast-css-loader` was removed in the webpack config. Can we delete it?

Reply:
yup